### ISSUE TITLE
More Installation Details for Python Novices

### ISSUE THREAD

**Issue by DarioS** [marcelm/cutadapt#208]
For someone who has never programmed in Python before, the installation instructions are hard to follow for an installation that does not have administrator privileges. I did

    pip install cutadapt -t /verona/biostat/sofware/Python

but I'm not sure what else to do to get it running at the command line.

**Comment by marcelm** [comment]
Have you seen [these instructions](http://cutadapt.readthedocs.io/en/stable/installation.html#quickstart)? They are explicitly made for users without administrator priviledges.

**Comment by DarioS** [comment]
Yes, but I didn't want to install it into the home directory, but another location on the server, where software shared amongst the group is stored. I did a `--user` install, then I used `cp -R` to move it to where I want it, because I wasn't sure how to use `pip` for that.

I don't see `bin` and `lib` folders when I use `-t`. Could you explain the correct way to do it, rather than `cp -R` of the `.local` folders, if it's not too hard?

**Comment by marcelm** [comment]
On the same page, just a few sections down, there’s a section about "shared installation". It says "on a cluster", but if you just follow the first instructions, it should work in your situation.

Here is the snippet you need:
```
BASE=/software/cutadapt-1.11
virtualenv $BASE/venv
$BASE/venv/bin/pip install --install-option="--install-scripts=$BASE/bin" cutadapt==1.11
```
You may want to adjust this to your needs.

You cannot easily move Python packages around after they are installed. You need to provide the correct paths at installation time. Two directories are relevant in the snippet above: First, the path to the virtualenv. This can be located anywhere. Second, the bin directory given to `--install-scripts`. Above, it’s within the $BASE directory, but that does not have to be the case. You can instead use your group’s common `bin` directory instead if you have that.

**Comment by DarioS** [comment]
Thank you. I will ask the system administrator to install `virtualenv` on the server.
